Update README.md (#1986)
amplication/amplication: README.md

## Development

Amplication is using a mono-repo with multiple packages. To initialize all the packages on a local development environment, follow these steps:

> First, execute this command, in the root folder of the projec:

Reviewer: project

Reviewer: First, execute this command in the project root folder

> First, execute this command, in the root folder of the projec:

Reviewer: ```suggestion
Execute the following commands in the project root folder:
```

```
npm install
```

> Then, execute this command, also in the root folder of the project:

Reviewer: ```suggestion
```

```
npm run setup:dev

> ```

Reviewer: Execute the following commands in the project root folder:
```
npm install
```
```
npm run setup:dev
```

> The second command will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.

Reviewer: This will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.

Reviewer: ```suggestion

This will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.
```
